app/models/mixins/supports_feature_mixin.rb

```diff
@@ -52,11 +52,46 @@ module SupportsFeatureMixin
   #
   extend ActiveSupport::Concern
 
+  QUERYABLE_FEATURES = {
+      :events => 'Query for events',
+      :control => '',
+      :live_migrate => '',
+      :evacuate => '',
+      :smartstate_analysis => '',
+      :retire => '',
+      :associate_floating_ip => '',
+      :disassociate_floating_ip => '',
+      :delete => '',
+      :resize => '',
+      :migrate => '',
+      :launch_cockpit => '',
+      :provisioning => '',
+      :reconfigure => '',
+      :regions => '',
+      :discovery => '',
```

Comment:
Minor, but a) alignment will make this much easier to read (looks like a table which the eyes play well with) and b) either alphabetize, or groups into related sections and alphabetize within those (makes finding things simpler).